flixel/system/FlxBGSprite.hx

```diff
@@ -20,6 +20,9 @@ class FlxBGSprite extends FlxSprite
 	@:access(flixel.FlxCamera)
 	override public function draw():Void
 	{
+		if (alpha == 0.0)
+			return;
+
```

Comment:
I see no reason for this class to check this when all other FlxSprites do not. This change will only serve to confuse people who expect this to behave exactly like other FlxSprites. I also would not want this change made in FlxSprite, I recommend either setting visible to false, manually to prevent draw calls, or making an extending class in your program that has this change

```suggestion
```

Reply:
also this wouldn't have had any effect, since the sprite's alpha is typically 1.0, even with a transparent bgColor. Changing the substate's bgColor only changes the sprite's pixel color